go/roothash/api/commitment/pool_test.go

```diff
@@ -1095,6 +1120,7 @@ func setupDiscrepancy(
 	sks []signature.Signer,
 	committee *scheduler.Committee,
 	nl NodeLookup,
+	enoughBackupCommits bool,
```

Comment:
:information_source: due to some overlap between the worker and backup committees, some tests for worker bad behavior also result in missing backup commits